daemon/list_test.go

```diff
@@ -16,7 +19,10 @@ import (
 	is "gotest.tools/v3/assert/cmp"
 )
 
-var root string
+var (
+	root    string
+	randGen *rand.Rand
```

Comment:
IMO, we shouldn't make that global and should keep it local to the test.

As soon as we have parallel tests sampling the random source (or maybe we have that already), the output won't be consistent because goroutines will sample it in a non-deterministic order.

Reply:
Thanks @vvoland, makes sense, done.